services/spar/src/Spar/Scim/User.hs

```diff
@@ -300,7 +300,6 @@ createValidScimUser (ValidScimUser user uref handl mbName richInfo) = do
     -- FUTUREWORK(arianvp): Get rid of manual lifting. Needs to be SCIM instances for ExceptT
     -- This is the pain and the price you pay for the horribleness called MTL
     storedUser <- lift $ toScimStoredUser buid user
-    idpConfig <-  lift $ SAML.getIdPConfigByIssuer (uref ^. SAML.uidTenant)
```

Comment:
I'm wondering if this changes behaviour? Are we sure this is the same `idpConfig` ? can there be cases where these are distinct? why not?

Reply:
I think except for yet another race condition (during the handling of this http request, the idp changed or went away, so the first lookup found something different from the second lookup) I don't think this is an issue.  I'll double-check tomorrow.

Reply:
b4a4dfca61545cee2ed983e5b02721d12ac352bd  good?